outlines/1.3-migration

```diff
@@ -0,0 +1,31 @@
+1. Breaking changes (as we'd normally do this)
+
+2. Recommended changes : Modules [link to app structure guide]
+  1. Applications: Move all files to imports, initialize in one spot, link via `import`
+  2. Import *all* symbols from packages, including meteor core packages.
```

Comment:
I don't know, it's kind of a rough spot overall.

On the one hand, I love the explicit imports.

On the other hand, it's weird that you don't actually get to see where the function came from, you just see where the _namespace_ came from, which kind of defeats the main purpose of imports.

On the third hand, it shuts up the linter complaining about globals... I wonder how hard it would really be to improve the APIs of all the core packages to be compatible with modules, so you could do `import { loginWithPassword } from 'accounts-password'`.

Speaking of which, do you currently still get globals from Meteor packages even when your file is in `imports`?